test/src/api/expect/testExpectAttribute.js

```diff
@@ -412,63 +412,63 @@ describe('expect.attribute', function() {
 
     Nocks.elementFound();
 
-    let expect = this.client.api.expect.element('#weblogin').to.have.attribute('class').equal('hp vasq').before(110);
+    const expect = this.client.api.expect.element('#weblogin').to.have.attribute('class').equal('hp vasq').before(160);
     Nocks.attributeValue(null)
       .attributeValue(null)
       .attributeValue(null)
       .attributeValue(null);
 
     return this.client.start(function(err) {
       assert.ok(err instanceof Error);
-      assert.strictEqual(expect.assertion.waitForMs, 110);
+      assert.strictEqual(expect.assertion.waitForMs, 160);
       assert.strictEqual(expect.assertion.passed, false);
       assert.ok(expect.assertion.retries > 1);
-      assert.ok(expect.assertion.message.startsWith('Expected element <#weblogin> to have attribute "class" equal to: "hp vasq" in 110ms - attribute was not found'));
+      assert.ok(expect.assertion.message.startsWith('Expected element <#weblogin> to have attribute "class" equal to: "hp vasq" in 160ms - attribute was not found'));
     });
   });
 
   it('to have attribute equal and custom waitFor [FAILED] - attribute not found', function() {
-    this.client.api.globals.waitForConditionPollInterval = 35;
-    this.client.api.globals.waitForConditionTimeout = 100;
+    this.client.api.globals.waitForConditionPollInterval = 100;
+    this.client.api.globals.waitForConditionTimeout = 250;
```

Comment:
why is this change necessary?

Reply:
It takes longer to run a test. Hence the timeouts have been increased to enable 1 retry.

Reply:
for some reason in windows, these tests are failing because these tests are time sensitive